eventmesh-metrics-plugin/eventmesh-metrics-prometheus/src/main/java/org/apache/eventmesh/metrics/prometheus/utils/PrometheusExporterUtils.java

```diff
@@ -70,7 +70,7 @@ public static <T extends Metric> void observeOfValue(Meter meter, String metricN
      * @return
      */
     public static String[] join(String metricName, String desc) {
-        return new String[] {metricName, desc};
+        return new String[]{metricName, desc};
```

Comment:
It is because the previous committer did not execute checkstyle, resulting in spotless not being processed successfully.

Checkstyle is a more relaxed requirement, and spotless will do more detailed formatting.

---

是因为之前的提交者, 没有执行过checkstyle, 导致spotless没有成功处理.
checkstyle是较为宽松的要求, spotless会做更精细化的格式化处理.